Formula/moar.rb

```diff
@@ -0,0 +1,22 @@
+class Moar < Formula
+  desc "Nice to use pager for humans"
+  homepage "https://github.com/walles/moar"
+  url "https://github.com/walles/moar/archive/refs/tags/v1.8.1.tar.gz"
+  sha256 "f4edbca2cce46e64d033536958d813ed70e7668ab52316cb87e42cccdb00eaaf"
+  license "BSD-2-Clause"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = "-s -w -X main.versionString=v#{version}"
+    system "go", "build", *std_go_args(ldflags: ldflags)
+  end
+
+  test do
+    # Test piping something through moar
+    (testpath/"test.file").write <<~EOS
+      tyre kicking
+    EOS
+    assert_equal "tyre kicking", shell_output("moar test.file").strip
```

Comment:
```suggestion
    (testpath/"test.txt").write <<~EOS
      tyre kicking
    EOS
    assert_equal "tyre kicking", shell_output("#{bin}/moar test.txt")
```
Sorry, forgot to add this suggestion last time. Should be good to merge once this is addressed.